pandas/tests/reshape/test_wide_to_long.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
```

Comment:
can you move existing wide_to_long tests from other reshaping tests to here (in test_melt)

Reply:
Thanks for all the help on my PR!

I made the style fixes and moved the test to the TestWideToLong class in test_melt.py. I then ran pytest locally and passed. Before updating the PR, I merged master changes from the past few days to my branch doing:
```git checkout master```
```git pull upstream master --ff-only```
```git checkout my-branch-name```
```git fetch upstream```
```git merge upstream/master```
There weren't any conflicts, but to be safe I figured I'd run test_melt.py again. That's when I got the following error:
```
(pandas-dev) Charless-MacBook-Air:reshape mcallistercs$ pytest test_melt.py 
Traceback (most recent call last):
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/_pytest/config/__init__.py", line 377, in _getconftestmodules
    return self._path2confmods[path]
KeyError: local('/Users/mcallistercs/Desktop/Github/pandas-csmcallister/pandas/tests/reshape/test_melt.py')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/_pytest/config/__init__.py", line 377, in _getconftestmodules
    return self._path2confmods[path]
KeyError: local('/Users/mcallistercs/Desktop/Github/pandas-csmcallister/pandas/tests/reshape')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/_pytest/config/__init__.py", line 408, in _importconftest
    return self._conftestpath2mod[conftestpath]
KeyError: local('/Users/mcallistercs/Desktop/Github/pandas-csmcallister/pandas/conftest.py')

During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/_pytest/config/__init__.py", line 414, in _importconftest
    mod = conftestpath.pyimport()
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/py/_path/local.py", line 668, in pyimport
    __import__(modname)
  File "/Users/mcallistercs/virtualenvs/pandas-dev/lib/python3.6/site-packages/_pytest/assertion/rewrite.py", line 226, in load_module
    py.builtin.exec_(co, mod.__dict__)
  File "/Users/mcallistercs/Desktop/Github/pandas-csmcallister/pandas/conftest.py", line 458, in <module>
    from hypothesis import strategies as st
ModuleNotFoundError: No module named 'hypothesis'
ERROR: could not load /Users/mcallistercs/Desktop/Github/pandas-csmcallister/pandas/conftest.py
```
I then opened a new shell and pip installed hypothesis, but still received the same error when trying to test again. Any help would be greatly appreciated!